### DISCUSSION TITLE
Relationship OAI / Magma

### DISCUSSION THREAD

**Discussion by koh-osug** [magma/magma#14373] [Q&A] [answered]
It seems Magma is using OAI as its core. What is the relationship between the projects? Magma has just grabbed the OAI code and since then the code base diverged? Or was code resubmitted to OAI?

**Comment by sebathomas** [comment, marked as answer]
There was a "history lesson" about Magma and OAI recently posted on Slack. See: https://magmacore.slack.com/archives/C018J8UMGMR/p1665131776239969?thread_ts=1664915985.071189&cid=C018J8UMGMR
Basically, Facebook tried to contribute back some of their changes to OAI but the code bases had diverged too much and it became impossible to reintegrate.

**Reply by koh-osug** [reply]
Thanks. Since this information is not in the public domain, I tried to join Slack on https://github.com/magma/community but the link is not working anymore. How can I join?

**Reply by sebathomas** [reply]
You can sign up via http://slack.magmacore.org/
